src/components/FetchWeatherData/FetchWeatherData.tsx

```diff
@@ -1,19 +1,50 @@
-import {useState, useEffect} from 'react';
-import {axiosDefaultConfig} from 'api/axiosDefaultConfig';
-import {LONDON_GEO} from './londonGeo';
+import { useState, useEffect } from 'react';
+import pick from 'lodash/pick';
+import { axiosDefaultConfig } from 'api/axiosDefaultConfig';
+import Location from 'components/LocationSelector/location';
 
-// TODO: Describe the type of data in interface below
-interface FetchedWeatherData {}
+interface Weather {
+  description: string;
+  icon: string;
+  main: string;
+}
+
+interface Main {
+  humidity: number;
+  temp: number;
+}
 
-export default function FetchWeatherData() {
-  const {lat, lon} = LONDON_GEO;
-  const [weatherData, setWeatherData] = useState<FetchedWeatherData | null>(
-    null,
-  );
+interface List {
+  dt_txt: string;
+  main: Main;
+  weather: Weather[];
+}
+
+interface City {
+  country: string;
+  name: string;
+}
+
+interface FetchedWeatherData {
+  city: City;
+  list: List[];
```

Comment:
We don't need to split this types into several interfaces. Combine these types in one interface